sor10874/readmes/25-30 (#725)
Esri/arcgis-runtime-samples-android: java/identify-layers/README.md

# Identify layers

Identify features in all layers in a map. 

![Image of identify layers](identify-layers.png)

## Use case

IdentifyLayers allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.

## How to use the sample

> Tap to identify features. An alert will show all layers with features under the cursor, as well as a layer count.

Reviewer: Sample does not display a layer count, but does display the number of features within each layer.

## How it works

> 1. The tapped position is passed to `MapView.IdentifyLayersAsync`

Reviewer: lower case method name
```suggestion
1. The tapped position is passed to `MapView.identifyLayersAsync`
```

2. For each `IdentifyLayerResult` in the results, features are counted. 
    * Note: there is one identify result per layer with matching features; if the feature count is 0, that means a sublayer contains the matching features.

## Relevant API

* IdentifyLayerResult

> * IdentifyLayerResult.LayerContent.Name

Reviewer: Remove method names in API section
```suggestion
```

> * IdentifyLayerResult.SublayerResults

Reviewer: ⬆️ 
```suggestion
```

> * MapView.IdentifyLayersAsync

Reviewer: ⏫ 
```suggestion
* MapView
```

## Additional information

> The GeoView supports two methods of identify: `identifyLayer()`, which identifies features within a specific layer and `identifyLayers()`, which identifies features for all layers in the current view.

Reviewer: it looks like both of these methods are only available in async form, but I'm not 100% sure. Check here https://developers.arcgis.com/android/latest/api-reference/reference/com/esri/arcgisruntime/mapping/view/GeoView.html#identifyLayerAsync(com.esri.arcgisruntime.layers.Layer,android.graphics.Point,double,boolean,int)

```suggestion
The GeoView supports two methods of identify: `identifyLayerAsync()`, which identifies features within a specific layer and `identifyLayersAsync()`, which identifies features for all layers in the current view.
```

Author: thanks Hudson, this is a good catch!
